File: Vincent OS/Avant.vb
Public Class Avant

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        Options.RadioButton1.Checked = True
        Démarrage.StartPosition = FormStartPosition.CenterScreen
        Démarrage.Show()
        Me.Hide()
    End Sub

    Private Sub Button2_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button2.Click
        Options.RadioButton2.Checked = True
        Démarrage.Show()
        If Démarrage.Visible = True Then
            Démarrage.FormBorderStyle = Windows.Forms.FormBorderStyle.None
            Arrêt.FormBorderStyle = Windows.Forms.FormBorderStyle.None
            Login.FormBorderStyle = Windows.Forms.FormBorderStyle.None
            Login2bureau.FormBorderStyle = Windows.Forms.FormBorderStyle.None
            Login2bureau2.FormBorderStyle = Windows.Forms.FormBorderStyle.None
            Bureau.FormBorderStyle = Windows.Forms.FormBorderStyle.None
            Bureau2.FormBorderStyle = Windows.Forms.FormBorderStyle.None
            Bureau.Width = My.Computer.Screen.Bounds.Width
            Bureau.Height = My.Computer.Screen.Bounds.Height
            Bureau2.Width = My.Computer.Screen.Bounds.Width
            Bureau2.Height = My.Computer.Screen.Bounds.Height
            Arrêt.Width = My.Computer.Screen.Bounds.Width
            Arrêt.Height = My.Computer.Screen.Bounds.Height
            Démarrage.Width = My.Computer.Screen.Bounds.Width
            Démarrage.Height = My.Computer.Screen.Bounds.Height
            Login.Width = My.Computer.Screen.Bounds.Width
            Login.Height = My.Computer.Screen.Bounds.Height
            Login2bureau.Width = My.Computer.Screen.Bounds.Width
            Login2bureau.Height = My.Computer.Screen.Bounds.Height
            Login2bureau2.Width = My.Computer.Screen.Bounds.Width
            Login2bureau2.Height = My.Computer.Screen.Bounds.Height
        End If
        Me.Hide()
    End Sub

    Private Sub Button3_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button3.Click
        alertelog.Show()
    End Sub
End Class

File: src/Vincent OS/Avant.cs
using System;
using System.Windows.Forms;

namespace Vincent_OS
{
    public partial class Avant
    {
        public Avant()
        {
            InitializeComponent();
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.Custom.RadioButton1.Checked = true;
            My.MyProject.Forms.Démarrage.StartPosition = FormStartPosition.CenterScreen;
            My.MyProject.Forms.Démarrage.Show();
            Close();
            GC.Collect();
            GC.WaitForPendingFinalizers();
        }

        private void Button2_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.Custom.RadioButton2.Checked = true;
            My.MyProject.Forms.Démarrage.Show();
            if (My.MyProject.Forms.Démarrage.Visible == true)
            {
                Restart restart = new Restart();
                // On change de mode de forme de la fenêtre
                My.MyProject.Forms.Démarrage.FormBorderStyle = FormBorderStyle.None;
                My.MyProject.Forms.Arrêt.FormBorderStyle = FormBorderStyle.None;
                My.MyProject.Forms.Login.FormBorderStyle = FormBorderStyle.None;
                My.MyProject.Forms.Login2bureau.FormBorderStyle = FormBorderStyle.None;
                My.MyProject.Forms.Login2bureau2.FormBorderStyle = FormBorderStyle.None;
                My.MyProject.Forms.Bureau.FormBorderStyle = FormBorderStyle.None;
                My.MyProject.Forms.Bureau2.FormBorderStyle = FormBorderStyle.None;
                restart.FormBorderStyle = FormBorderStyle.None;
                // On met une nouvelle localisation
                My.MyProject.Forms.Bureau.Width = My.MyProject.Computer.Screen.Bounds.Width;
                My.MyProject.Forms.Bureau.Height = My.MyProject.Computer.Screen.Bounds.Height;
                My.MyProject.Forms.Bureau2.Width = My.MyProject.Computer.Screen.Bounds.Width;
                My.MyProject.Forms.Bureau2.Height = My.MyProject.Computer.Screen.Bounds.Height;
                My.MyProject.Forms.Arrêt.Width = My.MyProject.Computer.Screen.Bounds.Width;
                My.MyProject.Forms.Arrêt.Height = My.MyProject.Computer.Screen.Bounds.Height;
                My.MyProject.Forms.Démarrage.Width = My.MyProject.Computer.Screen.Bounds.Width;
                My.MyProject.Forms.Démarrage.Height = My.MyProject.Computer.Screen.Bounds.Height;
                My.MyProject.Forms.Login.Width = My.MyProject.Computer.Screen.Bounds.Width;
                My.MyProject.Forms.Login.Height = My.MyProject.Computer.Screen.Bounds.Height;
                My.MyProject.Forms.Login2bureau.Width = My.MyProject.Computer.Screen.Bounds.Width;
                My.MyProject.Forms.Login2bureau.Height = My.MyProject.Computer.Screen.Bounds.Height;
                My.MyProject.Forms.Login2bureau2.Width = My.MyProject.Computer.Screen.Bounds.Width;
                My.MyProject.Forms.Login2bureau2.Height = My.MyProject.Computer.Screen.Bounds.Height;
                restart.Width = My.MyProject.Computer.Screen.Bounds.Width;
                restart.Height = My.MyProject.Computer.Screen.Bounds.Height;
            }
            Close();
            GC.Collect();
            GC.WaitForPendingFinalizers();
        }

        private void Button3_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.alertelog.Show();
        }
    }
}